### PR TITLE
feat(translation): add pt translation

### PR DESCRIPTION
**Pull request by Jpfonseca**
Added pt-pt as asked in #439 .

### CODE REVIEW

**Comment by lex111** [comment]
@Jpfonseca It's just Portuguese, right? If so, then leave just `pt`.

And please read the [instruction](https://github.com/gitpoint/git-point/blob/master/CONTRIBUTING.md#translations) for adding a new translation. Thanks in advance!

**Comment by Jpfonseca** [comment]
@lex111  there are 2 types of portuguese : pt-pt and pt-br

File: .all-contributorsrc [hunk 1]
```diff
@@ -381,6 +381,13 @@
         "code",
         "bug"
       ]
+    },
+    {
+      "login": "Jpfonseca",
```
Comment (housseindjirdeh) [inline .all-contributorsrc]: 🎉

**Comment by lex111** [comment]
@Jpfonseca I understood, but the fact is that we change the original name of the locale, making it compatible with [moment](https://github.com/moment/moment/) library, so we need the name of the locale to match the name of the locale in [moment](https://github.com/moment/moment/tree/develop/locale):

image

Therefore, please change the code with `pt-pt` everywhere to just `pt`.

P.S. @housseindjirdeh how correctly to describe this in the instruction?

**Comment by Jpfonseca** [comment]
@lex111  Sorry that i didn't do that before .

File: src/auth/screens/language-settings.js [hunk 2]
```diff
@@ -20,6 +20,11 @@ export default [
     name: 'Türkçe',
   },
   {
+    code: 'pt-pt',
```
Comment (lex111) [inline src/auth/screens/language-settings.js]: `pt` instead `pt-pt`.

**Comment by housseindjirdeh** [comment]
Cheers thank you @lex111 and sorry I didn't realize that earlier. Added that step to our [translations instructions](https://github.com/gitpoint/git-point/blob/master/CONTRIBUTING.md#translations).

**Comment by Jpfonseca** [comment]
@lex111  I think you can merge it now xD.
Sorry for the trouble

**Comment by housseindjirdeh** [comment]
@Jpfonseca my friend, no trouble at all! If anything, we need to apologize for not being super clear in the docs and having to tell you to change it up slghtly

Thank you for this, really means a lot to take the time to get all the translations in :heart:

**Pull request merged**